### ISSUE TITLE
Jupyter migration (meta issue)

### ISSUE THREAD

**Issue by jdfreder** [nteract/jupyter-display-area#17]
Things to consider:
- Moving this project to the Jupyter org.
- Implementing this repo (NOT transformime) in Typescript for the migration (like @ellisonbg suggested).
- If not implement this in Typescript, we definitely need d.ts files for it, if it is to be adopted by the main project
- Use it in the Phosphor notebook ???

**Comment by rgbkrk** [comment]
Gosh I thought Brian was suggesting transformime would be in typescript. I was thinking this package wouldn't expose much (if any) JavaScript externally beyond the web component definition.

**Comment by jdfreder** [comment]
Yes, Brian was suggesting both should be Typescript.  Although I still disagree on Transformime being Typescript, I do agree that this should be Typescript if it is to be adopted into the Jupyter org.
